File: WordApp/Models/User/UserProfileModel.cs
using System;
using Entities.Enums;
using WordApp.Models.Base;

namespace WordApp.Models.User
{
    public class UserProfileModel: BaseModel
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime? BirthDate { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public UserType UserType { get; set; }
    }
}

File: WordApp/ClientApp/src/app/models/users/user-profile.model.ts
export class UserProfileModel {
  public firstName: string;
  public lastName: string;
  public birthDate: Date;
  public name: string;
  public email:string;

  constructor(firstName: string, lastName: string, birthDate: Date) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.birthDate = birthDate;
  }
}
